src/OpenTK.Mathematics/Matrix/Matrix2.cs

```diff
@@ -367,19 +366,19 @@ public static Matrix2 Mult(Matrix2 left, float right)
         /// <param name="result">A new instance that is the result of the multiplication.</param>
         public static void Mult(ref Matrix2 left, ref Matrix2 right, out Matrix2 result)
         {
-            float lM11 = left.Row0.X,
-                lM12 = left.Row0.Y,
-                lM21 = left.Row1.X,
-                lM22 = left.Row1.Y,
-                rM11 = right.Row0.X,
-                rM12 = right.Row0.Y,
-                rM21 = right.Row1.X,
-                rM22 = right.Row1.Y;
+            float leftM11 = left.Row0.X,
```

Comment:
Don't declare multiple variables this way. Each one gets its own separate declaration.